pyproject.toml

```diff
@@ -1,3 +1,11 @@
+[[tool.mypy.overrides]]
+module = "fitz"
+ignore_missing_imports = true
+
+[[tool.mypy.overrides]]
+module = "bs4" # type exist for this but they are worse than useless
+ignore_missing_imports = true
```

Comment:
I tried using types with `bs4` but it just sucked so much

it was _"are you sure `element.href` maybe not be a list?"_ Because I guess you can defined html attributes more than once on the same element.